nx/lib/nx/type.ex

```diff
@@ -590,6 +590,30 @@ defmodule Nx.Type do
   """
   def to_string({type, size}), do: Atom.to_string(type) <> Integer.to_string(size)
 
+  @doc """
+  Returns the native binary representation the for smallest normal floating point number.
+
+  The smallest normal floating point number is the smallest positive value with
+  1 as the leading bit for the mantissa.
+  """
+  def smallest_normal({:bf, 16}) do
+    f32 = smallest_normal({:f, 32})
+
+    if System.endianness() == :little do
+      <<_::16, x::16-bitstring>> = f32
+      x
+    else
+      <<x::16-bitstring, _::16>> = f32
+      x
+    end
```

Comment:
there must be a better way to do this :p

Reply:
https://github.com/elixir-nx/nx/commit/fc9a2e7148a1d54e35fdd6edb38197c924d3e1d0